### PR TITLE
added cereal, expanded drop, added filter between columns

### PR DESCRIPTION
**Pull request by seankross**

### CODE REVIEW

File: src/components/blockly/dsl/datadsl.ts [hunk 1]
```diff
@@ -94,23 +96,72 @@ const dataDsl: BlockDomainSpecificLanguage = {
...
+                    name: "column3",
                 },
             ],
             previousStatement: DATA_SCIENCE_STATEMENT_TYPE,
             nextStatement: DATA_SCIENCE_STATEMENT_TYPE,
             // eslint-disable-next-line @typescript-eslint/no-explicit-any
             transformData: (b: BlockSvg, data: any[]) => {
-                const column = b.getFieldValue("column")
-                console.log("Drop: ", { column })
+                const columns = [1, 2, 3].map(column => { return b.getFieldValue(`column${column}`) })
```
Comment (pelikhan) [inline src/components/blockly/dsl/datadsl.ts:119]: Use lambda syntax for single expression functions

() => b.getField...

File: src/components/blockly/fields/BuiltinDataSetField.tsx [hunk 2]
```diff
@@ -6,6 +6,7 @@ import postLoadCSV from "../dsl/workers/csv.proxy"
 const builtins = {
     dummy: withPrefix("/datasets/dummy.csv"),
```
Comment (pelikhan) [inline src/components/blockly/fields/BuiltinDataSetField.tsx:7]: Should we erase dummy now?

File: src/workers/data/data.worker.ts [hunk 3]
```diff
@@ -29,9 +35,37 @@ const handlers: { [index: string]: (props: any) => object[] } = {
         return tidy(data, arrange(descending ? desc(column) : column))
     },
     drop: (props: DataDropRequest) => {
-        const { column, data } = props
-        if (!column) return data
-        else return tidy(data, select([`-${column}`])) 
+        const { columns, data } = props
+        if (!columns) return data
+        else return tidy(data, select(columns.map(column => { return `-${column}` })))
```
Comment (pelikhan) [inline src/workers/data/data.worker.ts:40]:
```
tidy(data, select(columns.map(column => { return `-${column}` }))) 
```
becomes
```
tidy(data, select(columns.map(column => `-${column}` ))) 
```

File: src/workers/data/data.worker.ts [hunk 4]
```diff
@@ -29,9 +35,37 @@ const handlers: { [index: string]: (props: any) => object[] } = {
...
-        if (!column) return data
-        else return tidy(data, select([`-${column}`])) 
+        const { columns, data } = props
+        if (!columns) return data
+        else return tidy(data, select(columns.map(column => { return `-${column}` }))) 
+    },
+    filter_columns: (props: DataFilterColumnsRequest) => {
+        const { columns, logic, data } = props
+        if (!columns) { return data }
+
```
Comment (pelikhan) [inline src/workers/data/data.worker.ts:45]: destructure the columns
```
const [left, right] = columns
```
then use left, right. Make sure the values of ``left`` and ``right`` are ok
```
if (!left || !right) return data
```

**Comment by pelikhan** [comment]
Looks good. merge when ready.

File: src/components/blockly/fields/BuiltinDataSetField.tsx (hunk 2 above)
Comment (seankross) [reply src/components/blockly/fields/BuiltinDataSetField.tsx:7]: I erased it and added `penguins`, a larger dataset that we could use in classrooms.

**Comment by pelikhan** [comment]
You forgot to add pinguins

**Review by pelikhan — APPROVED** [review]

**Comment by pelikhan** [comment]
Once you have a approval, feel free to squash and merge

**Pull request merged**